src/index.html

```diff
@@ -13,20 +13,29 @@
     <title>Document</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="/src/style.css"
```

Comment:
The path to the CSS file should be `href="src/style.css"` instead of `href="/src/style.css"` to ensure correct linking.